cmake/macros/macro_define_interface_target.cmake

```diff
@@ -78,9 +78,13 @@ function(define_interface_target _feature)
 
     if(DEFINED ${_feature}_VERSION)
       message(STATUS "    VERSION:             ${${_feature}_VERSION}")
-      set_target_properties(${_interface_target}
-        PROPERTIES VERSION "${${_feature_}_VERSION}"
-        )
+      # CMake versions prior to 3.19 have a significantly more restrictive
+      # set of allow interface target properties.
```

Comment:
allowed? allowing?

Reply:
allowed it should have been. :smile: Fixed.